wiki/FitNesseRoot/HsacAcceptanceTests/SlimTests/HttpTest/XmlHttpTest/content.txt

```diff
@@ -27,28 +27,34 @@ Incomplete address: no zip code</PO:confirmation>
-|show  |response                                                                                                                                      |
-|check |response status|500                                                                                                                           |
-|check |raw xPath      |/env:Envelope/env:Body/env:Fault/faultstring|Message does not have necessary info                                             |
-|show  |response headers                                                                                                                              |
-|check |response header|Content-length                              |556                                                                              |
-|note  |next line is commented out so we won't have a real exception, uncomment to see that SOAP content is shown in exception                        |
-|note  |check          |xPath                                       |/env:Envelope/env:Body/env:Fault/faultstring|Message does not have necessary info|
-|reject|post           |{{{<ServerFault/>}}}                        |to                                          |$url                                |
-|show  |response                                                                                                                                      |
-|check |response status|500                                                                                                                           |
-|check |raw xPath      |/env:Envelope/env:Body/env:Fault/faultstring|Message does not have necessary info                                             |
-|note  |next line is commented out so we won't have a real exception, uncomment to see that SOAP content is shown in exception                        |
-|note  |check          |xPath                                       |/env:Envelope/env:Body/env:Fault/faultstring|Message does not have necessary info|
-|reject|post           |{{{<NoXml/>}}}                              |to                                          |$url                                |
-|check |response status|200                                                                                                                           |
-|check |response       |<no xml                                                                                                                       |
-|note  |next line is commented out so we won't have a real exception, uncomment to see that non-XML content is shown in exception                     |
-|note  |check          |xPath                                       |count(/*)                                   |0                                   |
+|script   |xml http test                                                                                                                                       |
```

Comment:
Did you really mean to change this file? Or are all changes just left overs from when the post file to test was also part of this page?
If you meant no changes, please remove from pull request.